lib/wovnrb/store.rb

```diff
@@ -24,6 +24,7 @@ def self.default_settings
         'query' => [],
         'ignore_class' => [],
         'api_url' => 'https://wovn.global.ssl.fastly.net/v0/',
+        'api_url_versioned_widget' => 'https://wovn.global.ssl.fastly.net/widget',
```

Comment:
actually I think it would be good to do the same thing as WOVN.php and change `api_url` to be just 'https://wovn.global.ssl.fastly.net'. Then you can add the path as needed.
https://github.com/WOVNio/WOVN.php/pull/162/files